src/main/java/org/elasticsearch/indices/store/IndicesStore.java

```diff
@@ -318,38 +294,11 @@ public ClusterState execute(ClusterState currentState) throws Exception {
                         logger.trace("not deleting shard {}, the update task state version[{}] is not equal to cluster state before shard active api call [{}]", shardId, currentState.getVersion(), clusterState.getVersion());
                         return currentState;
                     }
-
```

Comment:
spoke to boaz about this and I would like to clean this class up in a different PR